### PR TITLE
Add missing release and charts labels in pods

### PR DESCRIPTION
**Pull request by javsalgar**

### CODE REVIEW

File: bitnami/zookeeper/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: zookeeper
-version: 0.0.3
+version: 0.0.4
```
Comment (tompizmor) [inline bitnami/zookeeper/Chart.yaml]: I don't see any other change in zookeeper, did you forget to commit it?

**Comment by carrodher** [comment]
Can you take advantage of this PR and remove `debian-9` from `appVersion` in the applications that contains this wrong parameter?

File: bitnami/zookeeper/Chart.yaml (hunk 1 above)
Comment (javsalgar) [reply bitnami/zookeeper/Chart.yaml]: That didn't require changes, reverting it.

File: bitnami/zookeeper/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: zookeeper
 version: 0.0.3
```
Comment (tompizmor) [inline bitnami/zookeeper/Chart.yaml]: now you should increase chart version because you are changing the app version

**Pull request merged**